packages/labs/observers/src/test/intersection_controller_test.ts

```diff
@@ -399,4 +399,38 @@ const canTest = () => {
     await intersectionComplete();
     assert.isTrue(el.observerValue);
   });
+
+  test('can observe changes when initialized after host connected', async () => {
```

Comment:
This test is fine, but the key test we need to add is that you can pass in a different `target` and verify that it correctly observes that target. (I think that test is needed for the other observers as well.)

Reply:
The `host` test was slightly covering this case, as `host` is assigned to `target` when  no target is passed in. For completeness and future proofing I've added an explicit target test as well to each controller that accepts a `target`.

Thank you!